include/mqt-core/dd/Package.hpp

```diff
@@ -1814,17 +1873,19 @@ template <class Config> class Package {
 
 private:
   /// TODO: introduce a compute table for the trace?
-  mCachedEdge trace(const mEdge& a, const std::vector<bool>& eliminate,
-                    std::size_t alreadyEliminated = 0) {
+  template <class OperandNode>
+  CachedEdge<OperandNode> trace(const Edge<OperandNode>& a,
+                                const std::vector<bool>& eliminate,
+                                std::size_t alreadyEliminated = 0) {
```

Comment:
Purely a consistency thing, but I think throughout most of the library, this template parameter is just referred to as `Node`

Reply:
I changed the code as suggested.